content/ux-strat-bootcamp/week-one/thursday/data-dashboard-analysis/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Data Dashboard Analysis
+content_type: project
+ready: true
+---
+
+A data dashboard is an information management tool that visually tracks, analyses and displays key performance indicators (KPI), metrics and key data points to monitor the health of a business, department or specific process.
+
+A wireframe is a simple visual guide that represents the skeletal framework of a digital product. 
+
+**Task**
```

Comment:
same feedback as before.

Headings should be added like this:

```
## Heading
```

So 
```
**Task**
```

Should be 

```
## Task
```